buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -1,6 +1,7 @@
 package org.tiqr.authenticator.buildsrc
 
-private const val kotlinVersion = "1.3.71"
+private const val kotlinVersion = "1.3.72"
+private const val navigationVersion = "2.3.0-alpha05"
```

Comment:
Do you need to use alpha versions?

Reply:
Not yet, but gonna need it to send result back to a fragment (which is only supported from version 2.3.0).